### PR TITLE
[ACS-8806] refactor process comments

### PR DESCRIPTION
**Pull request by g-jaskowski**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://hyland.atlassian.net/browse/ACS-8806

**What is the new behaviour?**
Process comments component reuses comments component instead of duplicated implementation.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by DenysVuika — APPROVED** [review]

File: lib/process-services/src/lib/process-list/components/process-list/process-list.component.ts [hunk 1]
```diff
@@ -85,6 +85,9 @@ export const processPresetsDefaultModel = {
     templateUrl: './process-list.component.html'
 })
 export class ProcessInstanceListComponent extends DataTableSchema implements OnChanges, AfterContentInit, PaginatedComponent {
+    @ViewChild(DataTableComponent)
```
Comment (MichalKinas) [inline lib/process-services/src/lib/process-list/components/process-list/process-list.component.ts:88]: Why do we need this change?

File: lib/process-services/src/lib/task-list/components/task-list/task-list.component.ts [hunk 2]
```diff
@@ -61,6 +73,9 @@ export const PRESET_KEY = 'adf-task-list.presets';
     styleUrls: ['./task-list.component.css']
 })
 export class TaskListComponent extends DataTableSchema implements OnChanges, AfterContentInit, PaginatedComponent, OnDestroy, OnInit {
+    @ViewChild(DataTableComponent)
```
Comment (MichalKinas) [inline lib/process-services/src/lib/task-list/components/task-list/task-list.component.ts:76]: Why do we need this change?

**Review by dominikiwanekhyland — APPROVED** [review]

File: lib/process-services/src/lib/task-list/components/task-list/task-list.component.ts (hunk 2 above)
Comment (g-jaskowski) [reply lib/process-services/src/lib/task-list/components/task-list/task-list.component.ts]: Reverted those changes, as I've found a possibly better solution

File: lib/process-services/src/lib/process-list/components/process-list/process-list.component.ts (hunk 1 above)
Comment (g-jaskowski) [reply lib/process-services/src/lib/process-list/components/process-list/process-list.component.ts]: Reverted those changes, as I've found a possibly better solution

**Review by swapnil-verma-gl — APPROVED** [review]

**Review by MichalKinas — APPROVED** [review]

**Pull request merged**